app/Http/Controllers/TaskCategoriesController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use Illuminate\Support\Facades\Validator;
+
+use App\Models\TaskCategory;
+
+class TaskCategoriesController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $categoryList = TaskCategory::all();
```

Comment:
Category should be user specific. So each user will have their own set of categories and they should be able to only use their own categories (not other user's)

Reply:
ok